README.md

```diff
@@ -135,6 +135,7 @@ Set up your GitHub Actions workflow with a specific version of your programming
 - [Get Commit Difference Count Between Two Branches](https://github.com/jessicalostinspace/commit-difference-action)
 - [Generate Release Notes Based on Git References](https://github.com/metcalfc/changelog-generator)
 - [Enforce Policies on GitHub Repositories and Commits](https://github.com/talos-systems/conform)
+- [Auto label issue based on a text/synonym](https://github.com/Renato66/auto-label)
```

Comment:
I do not see an explanation of the actions inputs in your README.md. I also think the README description is more descriptive than this title. Maybe changing it to "Auto Label Issue Based on Issue Description" may give more clarity? Also, please follow title casing requirements found here: https://github.com/sdras/awesome-actions/blob/master/contributing.md